portfolio/src/main/webapp/style.css

```diff
@@ -31,18 +21,41 @@ table.allTable{
     font-family: Tahoma, Geneva, sans-serif;
 }
 
-#socials-row{
+td#aboutMe-info{
+    font-size: 125%;
+}
 
+img#avatar{
```

Comment:
Avoid combining element selectors with id/class selectors unless absolutely necessary. In this case, you can just use #avatar

https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors